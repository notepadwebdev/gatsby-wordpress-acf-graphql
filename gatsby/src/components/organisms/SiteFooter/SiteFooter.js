import React from "react";
import "./SiteFooter.scss";

const SiteFooter = () => {
  return (
    <footer className="site-footer" role="contentinfo">
      <div className="container">
        <h2>Site Footer</h2>
      </div>
    </footer>
  );
};

export default SiteFooter;
