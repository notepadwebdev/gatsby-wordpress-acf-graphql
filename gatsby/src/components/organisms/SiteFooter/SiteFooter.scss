.site-footer {
  padding: 50px 0;
}
