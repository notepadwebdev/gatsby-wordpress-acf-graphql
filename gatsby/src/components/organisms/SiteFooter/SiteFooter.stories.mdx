import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
import SiteFooter from './SiteFooter';

<Meta
  title="Organisms/SiteFooter"
  component={SiteFooter}
/>

export const Template = args => <SiteFooter {...args} />;

export const Default = Template.bind({});
Default.parameters = {
  controls: { hideNoControlsWarning: true },
}

# SiteFooter

The main footer for the site.

<ArgsTable of={SiteFooter} />

## Stories
---

<Canvas>
  <Story name="Default" args={Default.args}>
    {Template.bind({})}
  </Story>
</Canvas>
