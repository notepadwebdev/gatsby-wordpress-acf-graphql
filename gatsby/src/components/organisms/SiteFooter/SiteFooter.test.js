import React from 'react';
import renderer from 'react-test-renderer';
import SiteFooter from './SiteFooter';

describe(`Site Footer component`, () => {
  it(`renders correctly`, () => {
    const component = renderer.create(<SiteFooter />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
