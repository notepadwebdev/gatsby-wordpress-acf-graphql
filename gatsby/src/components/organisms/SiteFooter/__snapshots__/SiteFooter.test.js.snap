// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Site Footer component renders correctly 1`] = `
<footer
  className="site-footer"
  role="contentinfo"
>
  <div
    className="container"
  >
    <h2>
      Site Footer
    </h2>
  </div>
</footer>
`;
